test/os/ui/filter/op/notlikeop.test.js

```diff
@@ -1,6 +1,6 @@
 goog.require('os.ui.filter.op.NotLike');
 
-describe('os.ui.filter.op.NotLike', function() {
+ddescribe('os.ui.filter.op.NotLike', function() {
```

Comment:
Double-d

Reply:
Thank you! Fixed. I wonder if we could scan for `ddescribe` and `iit` to fail CI/PR builds when present.